docs/index.md

```diff
@@ -27,22 +27,30 @@ And know how to use these tools:
 
 ## Introduction
 
+---
+
 The skeleton application is designed for rapid application testing and prototype development on the [NL Design System](http://storybook.nldesignsystem.nl/?path=/story/nl-design-system-introductie--page). There are several templates you can use as a running start - - each with their own functionality and purpose.
 
-_PIP_:
+_Website_:
 
-It provides a basic skeleton application with full [NL Design System](http://storybook.nldesignsystem.nl/?path=/story/nl-design-system-introductie--page) functionality that any developer can easily extend, view locally, and deploy to an online environment for demonstration purposes. It has [Common Gateway](https://github.com/CommonGateway) support, multiple navigations and much more.
+A simple website skeleton, including header, footers, routing and more. Does not initiate authentication. Choose this option without the need for a back end.
```

Comment:
"without the need for a back end" might give the notion that you _cannot_ actually add a back end. Might be better to state that this template does not automatically come with a gateway. A back end might be a too broad of a concept.